Focusgroup/explainer.md

```diff
@@ -705,7 +705,88 @@ We considered various alternative solutions before arriving at the current propo
     of building a native HTML feature to expose the built-in platform arrow-key
     navigation of certain controls.
 
-## 9. Open Questions
+## 9. Related Work
+
+### 9.1. CSS Basic UI 4 Keyboard Navigation properties
+
+[Since at least 2002](https://www.w3.org/TR/2002/WD-css3-ui-20020802/#nav-dir) the CSS WG
+has defined related CSS properties `nav-up`, `nav-right`, `nav-down`, `nav-left` in 
+[CSS Basic UI 4's Keyboard Control](https://drafts.csswg.org/css-ui-4/#keyboard). These 
+properties are expected to provide focus navigation control similar to `focusgroup`. They 
+differ in some significant ways:
+
+* Each of `nav-up`, etc., require an explicit content selector (id) for ordering, which
+    makes them relatively brittle. This design presents at least three challenges:
+    
+    * The possibility of mis-alignment to content. If stylesheet `nav-*` selectors are not
+        updated in terms of changes to document content (at authoring time) the navigation 
+        expectations can get out of sync.
+    * Unexpected side-effects with responsive design. If the wrapping and reflow of 
+        presentational content changes depending on device characteristics (such as orientation,
+        zoom level), authors must likely provide dynamic run-time updates to `nav-*` properties 
+        in order to keep top/left/bottom/right navigation logical per the current layout.
+    * Related to the previous point, it can be easy to make logical errors in navigation 
+        sequences when targetting specific directions. This can lead to directions that don't
+        match the property names (e.g., `nav-left` actually navigates up or right) while also
+        opening up the possibility of unidirectional navigation (e.g., after navigating right,
+        the user can't go "back" to the left due to missing or erroneous selectors).
+* Ordering is not based on content. These properties serve visual presentations, but possibly
+    make focus navigation illogical for accessibility users (especially when *any* element
+    can be targeted by selector.
+* Verbose descriptors. To get four-direction navigation on one element requires specifying 
+    four unique CSS properties.
+* Special focus handling. The use of a selector on one of the `nav-*` properties has the unique
+    property of making that element focusable upon keyboard navigation to it. Additional 
+    clarity on how this special semantic applies would be needed for implementation. It
+    is unclear (especially given the related Note in the spec) if this is a desirable 
+    behavior.
```

Comment:
Another point that could be made in how they differ is that `nav-*` properties require CSS, whereas `focusgroup` does not.

Reply:
I suppose that's true for Spatial Navigation as well in terms of [Controlling spatial navigation through declarative means](https://www.w3.org/TR/css-nav/#declarative).

Reply:
Thanks! I'll note that too.